js/data.js

```diff
@@ -7,7 +7,7 @@ var data = {
     fullName: '',
     location: '',
     avatarUrl: '',
-    bio: ''
-  },
-  entries: []
+    bio: '',
+    entries: []
```

Comment:
I don't know if this was a mistake, but you should keep this as a separate property from `profile`, since profile and entries are separate pieces of data :grin: